.rubocop.yml

```diff
@@ -14,3 +14,4 @@ RSpec/FilePath:
 Style/OptionalBooleanParameter:
   Exclude:
     - 'lib/easypost/shipment.rb'
```

Comment:
nit: Could probably remove the shipment line above.

Reply:
This line will be removed in the next PR when we remove `shipment.rb`